### PR TITLE
Stddev measure 177024189

### PR DESCRIPTION
**Pull request by ernestoarbitrio**

### CODE REVIEW

File: src/cr/cube/cubepart.py [hunk 1]
```diff
@@ -942,6 +942,19 @@ def summary_pairwise_indices(self):
             self, self._alpha, self._only_larger
         ).summary_pairwise_indices
 
+    @lazyproperty
```
Comment (slobodan-ilic) [inline src/cr/cube/cubepart.py:945]: `"t"` is before `"u"`

Comment (slobodan-ilic) [reply src/cr/cube/cubepart.py:945]: stddev before summary

File: src/cr/cube/matrix/measure.py [hunk 2]
```diff
@@ -551,7 +556,20 @@ class _Sums(_BaseSecondOrderMeasure):
     @lazyproperty
     def blocks(self):
         """2D array of the four 2D "blocks" making up this measure."""
-        return SumSubtotals.blocks(self._cube_measures.cube_sum.sums, self._dimensions)
+        return SumDiffSubtotals.blocks(
```
Comment (slobodan-ilic) [inline src/cr/cube/matrix/measure.py:559]: Is this on purpose? Do we want to change Sum -> SumDiff? Doesn't seem related to this PR.

Comment (ernestoarbitrio) [reply src/cr/cube/matrix/measure.py:559]: it is not related but i prefer to release it in this cr.cube version cause we don't want sum subtotals but we wanna have the subtotal differences.

File: src/cr/cube/matrix/cubemeasure.py [hunk 3]
```diff
@@ -161,6 +166,89 @@ def means(self):
...
+            else _CatXMrCubeStdDev
+            if dimension_types[1] == DT.MR
+            else _CatXCatCubeStdDev
+        )
+        return CubeSumsCls(
+            dimensions, cube.stddev[cls._slice_idx_expr(cube, slice_idx)]
+        )
+
+    @lazyproperty
+    def stddev(self):
```
Comment (slobodan-ilic) [inline src/cr/cube/matrix/cubemeasure.py:202]: Do we want to use `_base_values` instead of `stddev` for consistency?

File: src/cr/cube/matrix/measure.py (hunk 2 above)
Comment (ernestoarbitrio) [reply src/cr/cube/matrix/measure.py:559]: just avoid 2 releases .. that;s the only reason

File: src/cr/cube/matrix/cubemeasure.py (hunk 3 above)
Comment (ernestoarbitrio) [reply src/cr/cube/matrix/cubemeasure.py:202]: actually each BaseCubeMeasure have it's measure name property and not base_values. maybe i'm not following you. Can you explain your concern?

**Review by slobodan-ilic — APPROVED** [review]
